internal/app/api/api.go

```diff
@@ -74,6 +74,31 @@ func Init(conns *InfraConns) (http.Handler, error) {
 			method:  get,
 			handler: indexWebHandler.Index,
 		},
+		{
+			path:    "/customer",
+			method:  "POST",
+			handler: indexWebHandler.AddFriend,
+		},
+		{
+			path:    "/customer/{id}",
+			method:  "GET",
+			handler: indexWebHandler.FindFriendByID,
+		},
+		{
+			path:    "/customer",
+			method:  "PUT",
+			handler: indexWebHandler.UpdateFriend,
+		},
+		{
+			path:    "/customers",
+			method:  "GET",
+			handler: indexWebHandler.GetAllFriends,
+		},
+		{
+			path:    "/customer",
```

Comment:
which customer that we want to delete? missing {id}

Reply:
Updated